packages/projection-typeorm/src/entity/Handle.entity.ts

```diff
@@ -21,4 +21,9 @@ export class HandleEntity {
   @Column('varchar', { nullable: true })
   /** `null` when cardanoAddress === `null` */
   defaultForPaymentCredential?: Handle | null;
+  @OneToOne(() => HandleEntity, { nullable: true, onDelete: 'CASCADE' })
+  @JoinColumn()
+  parentHandle?: HandleEntity;
```

Comment:
I think we're still missing an update to `storeHandles` that sets this column. Also a migration in `cardano-services`.

Reply:
I can't do the migration because it needs the `local-network`. But I'll review this.

Reply:
You don't need local network to generate a migration, only postgres. See https://github.com/input-output-hk/cardano-js-sdk/tree/master/packages/cardano-services#generating-projector-schema-migrations

Reply:
True, I originally thought it did: 912f9ea8e9bf46eb2e26967492c795360ecffa5c